src/providers/RelayProvider.ts

```diff
@@ -133,6 +137,16 @@ export class RelayProvider implements TLProvider {
     }
   }
 
+  public async getMetaTxStatus(
+    identityAddress: string,
+    metaTransactionHash: string
+  ) {
+    const { MetaTransactionStatus } = await this.fetchEndpoint<any>(
```

Comment:
I would return the whole object here so that we can later use also different values returned by the relay.